examples/pxScene2d/src/rasterizer/pxRasterizer.cpp

```diff
@@ -1,10 +1,26 @@
 #include "pxTimer.h"
 #include "pxRasterizer.h"
 
-#include "xs_Core.h"
-#include "xs_Float.h"
+#ifdef XS_CODE_ENABLED
+#  include "xs_Core.h"
+#  include "xs_Float.h"
+#else
+#  include <cstdint>
+#  include <math.h>
+   inline int32_t xs_RoundToInt(const double d)
+   {
+     return floor(d + 0.5);
+   }
+
+   inline int32_t xs_CRoundToInt(const double d)
+   {
+     return ceil(d);
+   }
+#endif
```

Comment:
One comment related to this implementation:

- xs_RoundToInt() is implemented as per https://github.com/pxscene/pxCore/blob/master/examples/pxScene2d/src/rasterizer/xs_code/xs_Float.h#L403

- xs_CRoundToInt() original implementation is at https://github.com/pxscene/pxCore/blob/master/examples/pxScene2d/src/rasterizer/xs_code/xs_Float.h#L352

For `_xs_DEFAULT_CONVERSION != 0` both implementations looks the same, so why there were two different functions? I simply guessing that maybe there is an issue and assumed that `C` in `xs_CRoundToInt` stands for `ceil()` and implemented it in this way. If that assumption is entirely wrong, please let me know.

Reply:
Looks ok to move towards removal.